### PR TITLE
{bio}[foss-2016a,intel-2016a] R-bundle-Bioconductor-3.2 - rtracklayer update

### PR DESCRIPTION
**Pull request by verdurin**
`rtracklayer` has been updated to `1.30.3`.

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: ok to test

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 2 (2 easyconfigs in this PR)
node2002.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/49e48806c7dd17cdab88c7769369e353 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 12 out of 14 (2 easyconfigs in this PR)
node2721.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/239822ddc04c490e6ecbc9cdd29fb233 for a full test report.

File: easybuild/easyconfigs/r/R-bundle-Bioconductor/R-bundle-Bioconductor-3.2-foss-2016a-R-3.2.3.eb [hunk 1]
```diff
@@ -80,7 +80,7 @@ exts_list = [
     ('DEGseq', '1.24.0', bioconductor_options),
     ('hgu133plus2.db', '3.2.2', bioconductor_options),
     ('illuminaio', '0.12.0', bioconductor_options),
-    ('rtracklayer', '1.30.2', bioconductor_options),
+    ('rtracklayer', '1.30.3', bioconductor_options),
```
Comment (boegel) [inline easybuild/easyconfigs/r/R-bundle-Bioconductor/R-bundle-Bioconductor-3.2-foss-2016a-R-3.2.3.eb]: already updated to 1.30.4 by now...

We should probably do another full update round for all of these packages?

Comment (verdurin) [inline easybuild/easyconfigs/r/R-bundle-Bioconductor/R-bundle-Bioconductor-3.2-foss-2016a-R-3.2.3.eb]: Sure - started now.

**Comment by boegel** [comment]
@verdurin updates are required for

* limma -> 3.26.9
* edgeR -> 3.12.1
* Gviz -> 1.14.7
* Rsubread -> 1.20.5

other versions seem current

**Comment by verdurin** [comment]
@boegel - sure, I'll do those.

Why weren't they picked up by the test build, though?

I wonder whether we should somehow organise the constant Bioconductor updates, maybe checking every fortnight or so?

**Comment by boegel** [comment]
@verdurin which test build? Jenkins will not perform the actual installation, it only runs the unit tests (which check for missing deps, conflicts, etc.)

We could set up a regular check for Bioconductor, sure, or someone could convince them to stop trashing older versions...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2048.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/e76a17d29baaee6d4e0d5e1c56212046 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2603.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/79edd7cfa527f471aba9b7d6fc885dbf for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Going in, thanks @verdurin!

**Pull request merged**